langchain4j-mcp/src/main/java/dev/langchain4j/mcp/client/transport/http/StreamableHttpMcpTransport.java

```diff
@@ -132,7 +134,7 @@ public BodySubscriber<Void> apply(HttpResponse.ResponseInfo responseInfo) {
                             Optional<String> mcpSessionId =
                                     responseInfo.headers().firstValue("Mcp-Session-Id");
                             if (mcpSessionId.isPresent()) {
-                                log.debug("Assigned MCP session ID: " + mcpSessionId);
+                                log.debug("Assigned MCP session ID: {}", mcpSessionId);
```

Comment:
This logging statement is not related to traffic. Perhaps, should this be a separate logger than the one that logs raw requests and responses?